hugo/content/docs/ref/features.md

```diff
@@ -17,7 +17,7 @@ export class SNodeImpl extends SConnectableElementImpl implements Selectable, Fa
 }
 ```
 
-It is possible to fine-tune the behavior in the [dependency injection container]({{< ref "/docs/recipes/dependency-injection" >}}#the-container) by enabling or disabling features for a given model element type.
+It is possible to fine-tune the behavior in the [dependency injection container](../{{< ref "/docs/concepts/extension-points">}}#dependency-injection) by enabling or disabling features for a given model element type.
```

Comment:
this leads to build errors
`ERROR 2025/08/06 07:31:31 [en] REF_NOT_FOUND: Ref "/docs/concepts/extension-points": "/workspaces/sprotty-website/hugo/content/docs/ref/features.md:20:85": page not found`